data_diff/diff_tables.py

```diff
@@ -403,6 +404,15 @@ def _diff_tables(self, table1, table2, level=0, segment_index=None, segment_coun
             f"size: {table2.max_key-table1.min_key}"
         )
 
+        # When benchmarking, we want the ability to download the segments in
```

Comment:
I don't understand the word "parallel" in this context.

Wouldn't it be more correct to say "we want the ability to skip the checksum, and force the download of all segments" ?

Reply:
Sure.